tests/vagrant_inventory

```diff
@@ -0,0 +1,7 @@
+[vagrant]
+192.168.50.50
+
+[vagrant:vars]
+user=vagrant
+ansible_user=vagrant
+ansible_ssh_private_key_file=.vagrant/machines/default/libvirt/private_key
```

Comment:
aren't you missing a prefix? ~ perhaps?

Reply:
nope, it's in the same dir as the inventory